js/modules/k6/html/gen/gen_elements.go

```diff
@@ -232,8 +232,7 @@ func main() {
 	fs := token.NewFileSet()
 	parsedFile, parseErr := parser.ParseFile(fs, "elements.go", nil, 0)
 	if parseErr != nil {
-		log.Fatalf("warning: internal error: could not parse elements.go: %s", parseErr)
-		return
+		log.Fatalf("error: could not parse elements.go\n", parseErr)
```

Comment:
No need for \n here if you're using logrus, also use `log.WithError(parseErr).Fatal("...")`, etc. Same for all the other log statements.